src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$accent-blue-color: #00acdc;
+$primary-title-color: rgba(6, 11, 53, 1);
+$primary-grey-color: rgba(97, 96, 112, 1);
+$primary-white-color: rgba(255, 255, 255, 1);
+$border-color: rgba(243, 243, 243, 1);
```

Comment:
In my opinion, better to use hex-code for colors, because it's shorter and perhaps, more readable code. Also, i think that is not good practice, to make a new variable for every new color what you use in your project, it's just doing your code longer and less readable.

Reply:
I agree about the border-color variable, but other colors appear more than once.  About color formats, i think - that is not big problem, i copied styles from macket